graph/metafeeds_test.go

```diff
@@ -180,14 +181,14 @@ func (op PeopleOpAnnounceMetafeed) Op(state *testState) error {
 		return fmt.Errorf("wrong keypair type for mf: %T", mf.key)
 	}
 
-	// construct the announcement message according to spec 
-	// https://github.com/ssb-ngi-pointer/ssb-meta-feeds-spec#existing-ssb-identity
-	announcement := metamngmt.NewAnnounceMessage(kpMetafeed.ID())
-
-	// TODO: sign metafeedAnnounceMsg with metafeed keypair (note: not yet mentioned in spec)
-	_ = kpMain
+	// construct the announcement message according to spec
+	ma := legacy.NewMetafeedAnnounce(kpMetafeed.ID(), kpMain.ID())
```

Comment:
you and yr initialisms ;)

since you asked for a review: i'd prefer `announcement` and `signedAnnouncement`

Reply:
I think `ma` means mother

Reply:
Nope, wrong https://en.wikipedia.org/wiki/Ma_(goddess)